### ISSUE TITLE
Flameshot should check if the path is valid

### ISSUE THREAD

**Issue by mmahmoudian** [flameshot-org/flameshot#1139]
**Flameshot version**
Flameshot v0.8.3  (from `flameshot-git` AUR)
Compiled with Qt 5.15.0

**Describe the bug**
Flameshot fails to  save the captured screenshot the first time it is running, and produce the following error:
![image](https://user-images.githubusercontent.com/390889/98092889-dbb28b00-1e8f-11eb-85b2-c0db2e09906e.png)

From the second time on-wards, it asks for the path to save the image, I assume because the path it tries to save to does not exist.

**Note** that I do not have folder `Screenshots` in my `~/Pictures` folder, and I have never configured Flameshot to save images in such path:
![image](https://user-images.githubusercontent.com/390889/98093483-ae1a1180-1e90-11eb-8b7c-81a73d2b5403.png)
![image](https://user-images.githubusercontent.com/390889/98093518-b8d4a680-1e90-11eb-90e2-a91dc55495f6.png)

**To Reproduce**
`flameshot --gui` is mapped to my <kbd>PrntScrn</kbd>
1. When flameshot is not running (no tray icon, no process), I press the <kbd>PrntScrn</kbd>
2. select an area (with or without annotation)
3. click the floppy icon to save on disk
4. the error in the notification

**Expected behavior**
If the path does not exists, it warns the user and open a file browser window and ask the user where to save the captured image

**System Information**
Operating System: Manjaro Linux
KDE Plasma Version: 5.19.5
KDE Frameworks Version: 5.74.0
Qt Version: 5.15.1
Kernel Version: 5.8.16-2-MANJARO
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-4770K CPU @ 3.50GHz
Memory: 15.5 GiB of RAM
Graphics Processor: GeForce GTX 1060 6GB/PCIe/SSE2

**Comment by mmahmoudian** [comment]
I had to play wit this to figure out what was the issue: I realized that I was initiating the Flameshot with <kbd>PrntScrn</kbd> and then out of habit clicking on the tray icon to take screenshots. From there, it was clear that the keyboard shortcut setting was adding the `Pictures/Screenshots` as an argument. I went and checked and I was right:

![image](https://user-images.githubusercontent.com/390889/98121050-71154580-1eb7-11eb-879e-f1b4d290f719.png)


I still would suggest that the Flameshot checks if the provided path exists and if not, it shows proper warning and then open a file browser window and ask the user to choose the path for the image to be saved to.
